tests/test_dsl_pond.py

```diff
@@ -55,9 +60,9 @@ def test_pond_basic():
 
     with tempfile.TemporaryDirectory() as tempdir:
         tester.compile_and_run(target="verilator",
-                               directory=tempdir,
+                               directory="tempdir",
```

Comment:
Can you turn this back to the variable and get rid of the trace, otherwise this might fail if the directory doesn't exist

Reply:
Thanks for the catch! I forgot to turn the variable back after debugging